htdocs/index.html

```diff
@@ -69,6 +69,7 @@ <h2>How do I get rid of systemd?</h2>
 		<p>Alternatively you can use Linux distributions which do not use systemd by default:</p>
 
 		<ul class="distros">
+                        <li><a href="https://sulinos.github.io"><img src="https://gitlab.com/sulinos/look-and-feel/Sulin-Artworks/-/raw/master/sulin-logo.svg" alt="SulinOS" /></a></li>
```

Comment:
Could you please add the image file to the commit instead of hotlinking it? It would also be nice, if you could keep the indentation.

Reply:
this is sulinos logo